Lib/ast.py

```diff
@@ -667,6 +667,20 @@ def set_precedence(self, precedence, *nodes):
         for node in nodes:
             self._precedences[node] = precedence
 
+    def get_raw_docstring(self, node):
+        """If a docstring node is found in the body of the *node* parameter,
+        return that docstring node, None otherwise."""
+        if not isinstance(
```

Comment:
```suggestion
        """If a docstring node is found in the body of the *node* parameter,
        return that docstring node, None otherwise.
        
        A docstring is assumed to exist if the first node in a function, class or module node
        is a constant string literal (without assignments or otherwise).
        """
        if not isinstance(
```

Reply:
Why do we need to clarify the definition of 'docstring' here?

Reply:
Because is a "heuristic", no?

Reply:
If you think there is no way we can go into a false negative/positive, then don't commit the suggestion :)

Reply:
This part is how taken from `_PyAST_GetDocString`, which is the function compiler used to find docstrings. So I can't think of any false positives we can encounter.

Reply:
:ok_hand:

Reply:
Maybe then add a comment saying where the logic cames from if you don't mind

Reply:
Sure